asistente_ladm_col/lib/db/db_connector.py

```diff
@@ -346,6 +346,9 @@ def check_required_models(self, models):
 
         not_found = [model for model in models if not self.ladm_col_model_exists(model)]
 
+        if not_found == ['Modelo_Aplicacion_LADMCOL_Lev_Cat_V1_0']:
```

Comment:
Please add TODO that indicates that it should be removed in future releases

Reply:
Fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/577/commits/3c6ab9f70fa1deaa649e4e131be1baece41343a5